spring-integration-azure/src/test/java/com/microsoft/azure/spring/integration/SendOperationTest.java

```diff
@@ -84,7 +84,7 @@ public void testSendCreateSenderFailure() throws Throwable {
     }
 
     @Test
-    public void testSendFailure() {
+    public void testSendFailure() throws Throwable {
```

Comment:
Why change this?

Reply:
I changed this before for storage queue test to overwrite this method. I'll revert this change.